src/style.css

```diff
@@ -1 +1,208 @@
-/* add styles here */
+/* #region Reset Css */
+
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+font,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+search-bar,
```

Comment:
The `search-bar` selector is not a valid HTML tag. If this is meant to be a class, it should be prefixed with a dot (e.g., `.search-bar`).